src/main/kotlin/no/nav/familie/klage/kabal/domain/OversendtKlage.kt

```diff
@@ -82,6 +78,16 @@ enum class KildeFagsystem {
     KS
 }
 
+fun Fagsystem.tilKildeFagsystem() =
```

Comment:
Burde vi ha en egen fagsystem for klage i stedet, uten infotrygd sin? Så trenger vi ikke denne mappingen